components/src/FieldLabel/FieldLabel.story.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import {
+  FieldLabel,
+  Input,
+} from "ComponentsRoot";
+
+storiesOf("FieldLabel", module)
+  .add("FieldLabel", () => (
+    <FieldLabel labelText="Default label" />
+  ))
+  .add("with HelpText", () => (
+    <FieldLabel labelText="Default label" helpText="I am help text. I can give more details on the input below!" />
+  ))
+  .add("with RequirementText", () => (
+    <FieldLabel labelText="Default label" requirementText="(Required)" />
+  ))
+  .add("with all additional text", () => (
+    <FieldLabel labelText="Default label" helpText="I am help text. I can give more details on the input below!" requirementText="(Required)" />
+  ))
+  .add("with associated input", () => (
```

Comment:
Can we make it obvious that this is for a custom input? Maybe by just creating and using a custom input inline.